frontend/src/components/camp/picasso/Picasso.vue

```diff
@@ -0,0 +1,414 @@
+    },
+    isCategoryLoading (scheduleEntry) {
+      return scheduleEntry.activity().category()?._meta?.loading ?? false
+    },
+    intervalFormat (time) {
+      return this.$date.utc(time.date + ' ' + time.time).format(this.$tc('global.datetime.hourLong'))
+    },
+    dayFormat (day) {
+      if (this.$vuetify.breakpoint.smAndDown) {
+        return this.$date.utc(day.date).format(this.$tc('global.datetime.dateShort'))
+      } else {
+        return this.$date.utc(day.date).format(this.$tc('global.datetime.dateLong'))
+      }
+    },
+    weekdayFormat () {
+      return ''
+    },
+
+    entryClick ({ event: entry }) {
+      // if in read-only mode --> fire openEntry event
+      // in edit mode, the event is already fired by drag & drop logic
+      if (!this.editable) {
+        this.$emit('openEntry', entry)
+      }
+    }
+
+  }
+}
+</script>
+<style lang="scss">
+.ec-picasso {
+
+  .v-calendar-daily_head-day,
+  .v-calendar-daily__day {
+    min-width: 80px;
+  }
+
+  .v-event-timed {
+    padding: 1px;
+    font-size: 11px !important;
+    white-space: normal;
+    line-height: 1.15;
+    user-select: none;
+    -webkit-user-select: none;
+
+    .pl-1 {
+      padding-left: 2px !important;
+    }
+  }
+}
+
+.v-current-time {
+  height: 2px;
+  background-color: #ea4335;
+  position: absolute;
+  left: -1px;
+  right: 0;
+  pointer-events: none;
+
+  &::before {
+    content: '';
+    position: absolute;
+    background-color: #ea4335;
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+    margin-top: -5px;
+    margin-left: -6.5px;
+  }
+}
+
+.v-calendar .v-event-timed-container {
+  margin-right: 5px;
+}
+
+@media #{map-get($display-breakpoints, 'sm-and-up')}{
+  .ec-event--btn {
+    display: block !important;
+  }
+}
+
+.ec-event--btn {
+  padding: 0 !important;
+  min-width: 20px !important;
+  top: 0 !important;
+  right: 0 !important;
+  display: none;
+}
+
+.v-event-title {
+  hyphens: auto;
+  hyphenate-limit-chars: 6 3 3;
+  hyphenate-limit-lines: 2;
+  hyphenate-limit-last: always;
+  hyphenate-limit-zone: 8%;
+}
+
+.ec-daily_head-day-label {
+  font-size: 11px;
+  font-feature-settings: "tnum";
+  letter-spacing: -.1px;
+
+  .elipsis {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+    display: block;
+  }
+}
+</style>
+<style lang="scss" scoped>
+.v-card {
+  overflow: hidden;
+}
+
+::v-deep .v-calendar-daily__pane {
+  overflow-y: visible;
+}
+
+::v-deep .v-calendar-daily__scroll-area {
+  overflow-y: visible;
+}
+
+::v-deep .v-calendar-daily {
+  border-top: 0;
+  border-left: 0;
+  overflow-x: scroll;
+}
+
+::v-deep .v-calendar-daily__body {
+  overflow: visible;
+}
+
+::v-deep .v-event-timed.v-event--temporary {
+  border-style: dashed !important;
+  opacity: .8;
+}
+
+@media #{map-get($display-breakpoints, 'sm-and-up')}{
+  .v-event-timed {
+    &:hover .v-event-drag-bottom::after {
+      display: block;
+    }
+  }
+}
+
+.v-event-drag-bottom {
```

Comment:
Könnte man dieses CSS das klar nur mit Drag&Drop zu tun hat auch noch ins Composable auslagern?

Reply:
Wär mir grad keine gute Option bekannt. In den composables kann man keine styles definieren. Wenn man template+styles kapseln will, brauchts in der Regel eine eigene Komponente.